sdk/identity/identity/test/public/node/azurePipelinesCredential.spec.ts

```diff
@@ -4,7 +4,9 @@
 import { AzurePipelinesCredential } from "../../../src";
 import { isLiveMode } from "@azure-tools/test-recorder";
 import { assert } from "@azure-tools/test-utils";
-
+import { logger } from "@azure/keyvault-keys";
```

Comment:
Is keyvault-keys the appropriate place to import logger from? Seems like a typo and I don't expect identity tests to depend on another service package.

Reply:
No it should be @azure/logger. It's a typo.